src/components/journeys/driver-journey/driver-journey.jsx

```diff
@@ -20,6 +20,19 @@ const DriverJourney = ({configProps, driverObj}) => {
                             callback();
                         }
                         const portalData = [];
+
+                        const closeButton = document.getElementsByClassName('driver-popover-close-btn')[0];
+                        if (closeButton) {
+                            closeButton.textContent = '';
```

Comment:
Are these element property changes necessary because `driver.js` is setting up the button in a way that we can't fully control?

Reply:
@cwillisf it is using for the cross sign the sign for multiplication and when the browsers running on mac try to render it is it not centered